### PR TITLE
Add note to documentation for Ubuntu generate_data not running correctly

### PR DESCRIPTION
**Pull request by ocshawn**
### Improvements
- Add note to documentation for Ubuntu generate_data not running correctly

### CODE REVIEW

File: devHelper/README.md [hunk 1]
```diff
@@ -12,6 +12,10 @@ In the root directory of this repo, run the following command:
 ```
 sh ./generate_data.sh
 ```
+On Ubuntu may need to run directly
+```
+./generate_data.sh
```
Comment (mfshao) [inline devHelper/README.md:17]: i think in nowadays this is a more regular way to run script files, so maybe just replace `sh ./generate_data.sh` with this

Comment (mfshao) [reply devHelper/README.md:17]: @ocshawn still want to update this one?

Comment (ocshawn) [reply devHelper/README.md:17]: yes would still like to update this

Comment (ocshawn) [reply devHelper/README.md:17]: sorry just saw your note, i dont run things on mac so i dont know if it will work without the sh

Comment (mfshao) [reply devHelper/README.md:17]: yeah it can run on recent MacOSs so we are safe to update it

File: devHelper/README.md [hunk 2]
```diff
@@ -12,6 +12,10 @@ In the root directory of this repo, run the following command:
...
+```
```
Comment (ocshawn) [inline devHelper/README.md:18]:
```suggestion
./generate_data.sh
```
```

File: devHelper/README.md (hunk 1 above)
Comment (ocshawn) [inline devHelper/README.md:17]:
```suggestion
./generate_data.sh
```

**Review by mfshao — APPROVED** [review]

**Pull request merged**